components/CheckRepository.vue

```diff
@@ -72,7 +72,14 @@
               Check another?
             </a>
           </div>
-          <div v-else-if="processing" class="Processing">Processing...</div>
+          <div v-else-if="processing" class="Processing">
+            <scale-loader
+              :loading="processing"
+              :color="loaderOpt.color"
```

Comment:
```suggestion
              :color="$colorMode.value === 'dark' ? loaderOpt.darkColor : loaderOpt.lightColor"
```

Reply:
Sorry I forgot, 
I've implement those changes in the latest commit! Thanks!